doc/classes/OmniLight3D.xml

```diff
@@ -14,8 +14,8 @@
 	</tutorials>
 	<members>
 		<member name="omni_attenuation" type="float" setter="set_param" getter="get_param" default="1.0">
-			The light's attenuation (drop-off) curve. A number of presets are available in the [b]Inspector[/b] by right-clicking the curve. Zero and negative values are allowed but can produce unusual effects.
-			[b]Note:[/b] Very high [member omni_attenuation] values (typically above 10) can impact performance negatively if the light is made to use a larger [member omni_range] to compensate. This is because culling opportunities will become less common and shading costs will be increased (as the light will cover more pixels on screen while resulting in the same amount of brightness). To improve performance, use the lowest [member omni_attenuation] value possible for the visuals you're trying to achieve.
+			The parameter controls the distance attenuation function for omnilights. A value of 0 smoothly attenuates light at the range edges. Setting it to 1 approaches a physical lighting model but may result in distant objects receiving minimal light, even within the range. For instance, with a range of 4096, an object at 100 units receives less than 0.1 energy. A value of 0.5 approximates linear attenuation. Using negative or higher values may lead to unpredictable outcomes.
```

Comment:
I cannot speak on factual accuracy but for anything else I may:
- Surround the values with `[code]`;
- No need to start with "The paramenter", plus it's not consistent.
- "For instance" -> "For example".
```suggestion
			Controls the distance attenuation function for omnilights. A value of [code]0[/code] smoothly attenuates light at the range edges. Setting it to [code]1[/code] approaches a physical lighting model but may result in distant objects receiving minimal light, even within range. For example, with a range of [code]4096[/code], an object at [code]100[/code] units receives less than [code]0.1[/code] energy. A value of [code]0.5[/code] approximates linear attenuation. Using negative or higher values may lead to unpredictable outcomes.
```
As for the overall structure of the description, I'm not sure. It is pretty lengthy and could either be shortened or at the very least split onto a new line.

Reply:
(congrats on, if I'm not mistaken, first "official" review)